webpack.config.js

```diff
@@ -36,9 +40,15 @@ module.exports = {
     resolve: {
         modules: [
             path.resolve(__dirname, "socialhome"),
+            path.resolve(__dirname, "mocha/fixtures"),
```

Comment:
Now these are probably wrong - and probably unnecessary if the tests are working even if this is wrong ;)